### PR TITLE
use string constants as needed, use IEC notation

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: cli/delete.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ var (
 		},
 		cli.StringFlag{
 			Name:  "obj.size",
-			Value: "1KB",
+			Value: "1MiB",
```
Comment (klauspost) [inline cli/delete.go]: There shouldn't be a need to upload this big objects since it is only for delete tests.

Comment (harshavardhana) [reply cli/delete.go]: The reason I did that was for faster uploads @klauspost it's too slow.

Comment (klauspost) [reply cli/delete.go]: @harshavardhana How is increasing the size helping that?

**Comment by harshavardhana** [comment]
I changed the PR to not change the default obj.size for DELETE as I saw that indeed 1KiB is faster than 1MiB due to lower concurrency used in staging uploads. 

Instead, I moved warp outputs to IEC notation since we use 1024 as the divisor and also use humanize package to parse inputs for obj.size

**Comment by harshavardhana** [comment]
That was just based on this

**Pull request merged**